### PR TITLE
[ClassLoader] Deprecated the component

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Follow up of #20668

### CODE REVIEW

File: UPGRADE-3.3.md [hunk 1]
```diff
@@ -4,8 +4,7 @@ UPGRADE FROM 3.2 to 3.3
 ClassLoader
 -----------
 
- * The ApcClassLoader, WinCacheClassLoader and XcacheClassLoader classes have been deprecated
-   in favor of the `--apcu-autoloader` option introduced in composer 1.3
+ * The component is deprecated and will be removed in 4.0. Use composer instead.
```
Comment (javiereguiluz) [inline UPGRADE-3.3.md]: `composer` -> `Composer`

File: UPGRADE-4.0.md [hunk 2]
```diff
@@ -4,8 +4,7 @@ UPGRADE FROM 3.x to 4.0
 ClassLoader
 -----------
 
- * The ApcClassLoader, WinCacheClassLoader and XcacheClassLoader classes have been removed
-   in favor of the `--apcu-autoloader` option introduced in composer 1.3
+ * The component has been removed. Use composer instead.
```
Comment (javiereguiluz) [inline UPGRADE-4.0.md]: `composer` -> `Composer`

File: src/Symfony/Component/ClassLoader/CHANGELOG.md [hunk 3]
```diff
@@ -4,8 +4,7 @@ CHANGELOG
 3.3.0
 -----
 
- * The ApcClassLoader, WinCacheClassLoader and XcacheClassLoader classes have been deprecated
-   in favor of the `--apcu-autoloader` option introduced in composer 1.3
+ * deprecated the component: use composer instead
```
Comment (javiereguiluz) [inline src/Symfony/Component/ClassLoader/CHANGELOG.md]: `composer` -> `Composer`

**Review by javiereguiluz — APPROVED** [review]
👍

`composer` should be `Composer` in every deprecation message.

**Comment by stof** [comment]
👍

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**